le-parole-della-pubblica-amministrazione/l.rst

```diff
@@ -11,4 +11,11 @@ L
 
            *Testo definitivo della Legge di Bilancio 2018 (Legge 205/2017)*
    
-      
+    Locazione
+         Preferisci l'uso di termini più semplici (affitto)
+         
+         **Usa**
+            *...l'immobile viene affittato a partire da..."
```

Comment:
Grazie @Giacomomason per la pull request. 
C'è un piccolo errore formale alle righe 18 e 21. In RST, il singolo asterisco indica il corsivo. In queste due righe, il corsivo si apre ma non viene chiuso. 
Le virgolette alla fine delle frasi dovrebbero essere sostituite da un asterisco, altrimenti la compilazione risulterebbe in un errore.